packages/core/src/access/index.ts

```diff
@@ -0,0 +1,20 @@
+import { BaseListTypeInfo } from '../types';
+import { BaseAccessArgs } from '../types/config/access-control';
+
+export function allowAll() {
+  return true;
+}
+export function denyAll() {
+  return false;
+}
+
+export function allOperations<ListTypeInfo extends BaseListTypeInfo>(
+  func: (args: BaseAccessArgs<ListTypeInfo>) => boolean
+) {
```

Comment:
@mitchellhamilton or @dcousens This type functions but getting here felt like flailing not knowledge, and would love some scrutiny and feedback of this.